### DISCUSSION TITLE
VUCC Gridsquares Map & Grid Counts

### DISCUSSION THREAD

**Discussion by mwimages** [magicbug/Cloudlog#514] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**

I am a 100% portable station and frequent rover with several Station Profiles. When I initially uploaded my full log into Cloudlog (from every operating location, including those that are outside of my 200km VUCC circle boundary), I experienced the following two problems:

1.) The Analytics > Gridsquares map displayed every grid worked from all my Station Profiles, regardless of whether I worked them from locations outside my VUCC boundary. I love that this map is publicly viewable, but not that it gave an inaccurate view of the grids I needed.
2.) The Awards > VUCC counts only counted the grids worked from the Active Profile. There was no way to see the count from all Station Profiles within my VUCC boundary.

Currently, my workaround is to log only those stations that I work within my VUCC boundary, but I'd like to use Cloudlog while I'm out roving as well.

**Describe the solution you'd like**

Can we add a toggle in each Station Profile to indicate if the profile location is within my VUCC boundary? This could then be used to limit the Analytics > Gridsquares map to just those that users have worked within the rules of their VUCC award as well as to offer a complete count of worked grids somewhere on the Awards > VUCC screen.

**Describe alternatives you've considered**

Another solution would be the ability to create Awards Profiles similar to how awards are set up through LoTW. Users would assign certain rules to their award profiles (callsigns, date ranges, gridsquares, etc.). This would be an awesome way to allow having more than one VUCC award count, which I believe is an option that no other logging software (besides LoTW) has implemented.

**Additional context**

A similar solution could be used for other awards that limit operating zones. (WAS, etc.)

**Comment by Eric-Sparks** [comment]
I came here looking for the same thing.  I _think_ the solution would be to do what LoTW has done and allow the user to create a VUCC award group that encompasses the station locations that are within the VUCC award area that the user has created and also what bands they want to track in that particular award (VUCC worked-from areas differ between VHF/UHF and microwave).

Honestly, I can see doing this for all of the awards where working from a specific location is part of the criteria.  You could just create a default award, as is done now, but then allow the user to pull together station locations that meet their criteria.

**Comment by magicbug** [comment]
Best option for this is to use Staiton Logbooks these can have multiple locations and use that for grouping together things for awards thats the very reason it was built out
